src/components/DatePicker/CalendarPicker/index.tsx

```diff
@@ -142,10 +143,12 @@ function CalendarPicker({
     const hasAvailableDatesNextMonth = startOfDay(new Date(maxDate)) > endOfMonth(new Date(currentDateView));
     const hasAvailableDatesPrevMonth = endOfDay(new Date(minDate)) < startOfMonth(new Date(currentDateView));
 
+    const webOnlyMarginStyle = getPlatform(true) === CONST.PLATFORM.WEB ? {marginHorizontal: 4} : {};
+
```

Comment:
is getPlatform(true) correct here?

Reply:
yes, only the web version needs this padding
Here I'm avoiding web mobile as well. The only thing I'd add is to avoid applying it to shouldUseNarrowLayout as well